Makefile

```diff
@@ -22,11 +20,18 @@ OPERATOR_SDK_EXTRA_ARGS ?= --debug
 # test namespace name
 TEST_NAMESPACE ?= default
 
+# test parallelism
+TEST_PARALLEL ?= 8
```

Comment:
not sure 8 is good for travis or other localhost envs, why not to use the `-p` flag?, that reads like:

>  -p	Run in parallel with auto-detected number of nodes

Reply:
Citing the Ginkgo CLI: `The number of nodes used with -p is runtime.NumCPU() if runtime.NumCPU() <= 4, otherwise it is runtime.NumCPU() - 1 based on a rigorous science based heuristic best characterized as “my gut sense based on a few months of experience”`

My opinion here is that the CPU usage coming from the tests themselves is low. The real work happens outside of the go processes in the k8s cluster. How much load that cluster can handle is something I do not know how to determine programmatically.

Reply:
Discussed with @qu1queee; this setting is mainly for Travis where kind runs on the same machine, therefore will switch to `-p`. One can still run ginkgo directly to pass in any `-n` value.

Reply:
Done, see https://github.com/redhat-developer/build/pull/163/commits/7ee9f0765f5f92753d685a0c5ca7efed0be5665d#diff-b67911656ef5d18c4ae36cb6741b7965R29